src/WaferProber-Accretech_UFseries/main.py

```diff
@@ -261,140 +313,212 @@ def unconfigure(self):
         self.prober.z_down()
 
         if self.sweep_value_wafer != "Current wafer":
-            # Terminating the lot process bringing all wafers back to cassette
-            if hasattr(self, "is_run_stopped") and self.is_run_stopped():
-                message_box("Lot process will be finished which takes 1-2 minutes.\n\n"
-                            "Do not terminate the run and please wait until the program finishes."
-                            "You can close this message when the run has completed normally.")
 
-            self.prober.terminate_lot_process_immediately()
+            if self.exception_step_during_apply != "":
+                message_box("Accretech wafer prober failed to change to the next %s. Wafer remains on the chuck."
+                            % self.exception_step_during_apply.lower(), blocking=False)
+            else:
+                # Terminating the lot process bringing all wafers back to cassette
+                if hasattr(self, "is_run_stopped") and self.is_run_stopped():
+                    message_box("Lot process will be finished which takes 1-2 minutes.\n\n"
+                                "Do not terminate the run and please wait until the program finishes."
+                                "You can close this message when the run has completed normally.", blocking=False)
+
+                self.prober.preload_specified_wafer(9, 99)
+
+    def signout(self):
+        pass
 
     def apply(self):
```

Comment:
The apply() function is huge and might benefit from a docstring that summarized what is happening in the function.
The comments within the code are already helpful, but don't give a bigger picture that might help when looking at the driver in half a year.

Reply:
added a doctstring